src/parser/jobs/blm/RotationWatchdog.js

```diff
@@ -136,34 +207,48 @@ export default class RotationWatchdog extends Module {
 	}
 
 	_stopRecording() {
-		if (this._inFireRotation) {
+		if (this._inRotation) {
 			this._lockedBuffs = false
-			this._inFireRotation = false
+			this._inRotation = false
 			this._rotation.end = this.parser.currentTimestamp
 			// TODO: Use a better trigger for downtime than transpose
 			// TODO: Handle aoe things
 			// TODO: Handle Flare?
+			const MP = this._MP
```

Comment:
This is a completely redundant declaration and can just be inlined into the 1 use it has